cli/util/options.js

```diff
@@ -16,12 +16,11 @@ const colorsUtil = require("./colors");
 // S    | string array
 
 /** Parses the specified command line arguments according to the given configuration. */
-function parse(argv, config) {
+function parse(argv, config, populateDefaults = true) {
```

Comment:
Perhaps more clearly will be using "propagateDefault" wdyt?

Reply:
I agree with @MaxGraey.